modules/fx/arrow-fx-rx2/src/test/kotlin/arrow/fx/SingleKTests.kt

```diff
@@ -66,50 +69,61 @@ class SingleKTests : RxJavaSpec() {
     }
 
     "Multi-thread Singles finish correctly" {
-      val value: Single<Long> = SingleK.fx {
-        val a = Single.timer(2, TimeUnit.SECONDS).k().bind()
-        a
-      }.value()
-
-      val test: TestObserver<Long> = value.test()
-      test.awaitDone(5, TimeUnit.SECONDS)
-      test.assertTerminated().assertComplete().assertNoErrors().assertValue(0)
+      forFew(10, Gen.choose(10L, 50)) { delay ->
```

Comment:
Could not reproduce even when increased to 1000 tries.

Reply:
I think we could close #1776 and reopen if we see this happening again after your changes 👍